tests/framework/validators/CompareValidatorTest.php

```
@@ -37,364 +43,840 @@ protected function tearDown(): void
+        $validator->validateAttribute($model2, 'attr_test');
+
+        self::assertTrue(
+            $model2->hasErrors('attr_test'),
+            "Validation should fail when '50' is not > '75'.",
+        );
+    }
+
+    public function testClientValidateAttributeWithClosureCompareValue(): void
+    {
+        $closureConfig = [
+            'compareValue' => static fn(FakedValidationModel $model, string $attribute): string => 'closure_value',
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$attribute'. (undefined)

(UnusedFormalParameter)